docs/release-notes.md

```diff
@@ -1,5 +1,23 @@
 # Firefox Lockwise for Desktop Release Notes
 
+## 2.2.3-alpha
+
+_Date: 2019-05-31_
+
+### What's New
+
+* Lockwise is now _**localized**_ for a number of languages!
+
+### What's Fixed
+
+* If one of your logins has an unexpected website address, Lockwise no longer stops working and instead displays the address as-is.
+
+### Known Issues
+
+* This extension is designed to work with Firefox release 67 or higher.  There are reports of it installing then crashing version 66.  This issue is tracked in ([BMO 1555734](https://bugzilla.mozilla.org/show_bug.cgi?id=1555734)).
```

Comment:
Nit: technically 66.0.5, I'm not sure QA tested any other versions

Reply:
I'll change it to "certain versions of 66"